Code/Framework/AzFramework/AzFramework/Archive/Archive.cpp

```diff
@@ -2140,7 +2148,7 @@ namespace AZ::IO
             }
 
             currentDirPattern = currentDir + AZ_FILESYSTEM_SEPARATOR_WILDCARD;
-            currentFilePattern = currentDir + AZ_CORRECT_FILESYSTEM_SEPARATOR_STRING + "levels.pak";
+            currentFilePattern = currentDir + AZ_CORRECT_FILESYSTEM_SEPARATOR_STRING + "level.pak";
```

Comment:
nit:
Save a '+' operation, AZ_CORRECT_FILESYSTEM_SEPARATOR_STRING is a #define, 

```suggestion
            currentFilePattern = currentDir + AZ_CORRECT_FILESYSTEM_SEPARATOR_STRING "level.pak";
```